hlxsites/merative2
# fix(article-body): Adjust article body to align with design specs

## Issue

Fixes https://github.com/hlxsites/merative2/issues/195
  
## Description
- This PR adds style changes to the blog/article body to match the design specs.
- `Pullquote` block now displays the styles from the design
- `Featured Solution` block has been added to style the listing - https://fix-195-article-body-style--merative2--proeung.hlx.page/block-library/blocks/featured-solution

## Design Specs
- Figma - https://www.figma.com/file/dFJwrp6VkUMSirF2KMLRMx/Thought-leadership-library?type=design&node-id=357%3A17984&t=iaNohDrbt4220FLs-1

## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/drafts/putra/test-blog-detail-page
- After: https://fix-195-article-body-style--merative2--proeung.hlx.page/drafts/putra/test-blog-detail-page

## Testing Instruction

- Visit this test blog page - https://fix-195-article-body-style--merative2--proeung.hlx.page/drafts/putra/test-blog-detail-page
- Scroll down and ensure that the `Pull-quote` block matches the design
- Scroll down and ensure that the `Featured Solution` block matches the design
- Ensure that the Video embed includes a title

## Notes

- We'll need to update the content entry enablement to ensure that Content Authors understand the different blocks and text formats they should be using in the [SharePoint doc](https://merative.sharepoint.com/:w:/r/sites/Merative.comWebsite/_layouts/15/Doc.aspx?sourcedoc=%7B471AE649-EB64-4BDD-9961-E1CC011F020A%7D&file=test-blog-detail-page.docx&action=default&mobileredirect=true). @keith-kaplan Can you let me know if we have a process for updating Authors on changes?
- This PR is based on the `main` branch instead of `feat/thought-leadership` because these styles are part of the Blog and can be released before the Thought Leadership work has been completed.
